solr/solrj/src/java/org/apache/solr/client/solrj/impl/SolrClientBuilder.java

```diff
@@ -101,13 +116,27 @@ public B withConnectionTimeout(int connectionTimeoutMillis) {
    * sockets.
    *
    * <p>For valid values see {@link org.apache.http.client.config.RequestConfig#getSocketTimeout()}
+   *
+   * <p>* @deprecated Please use {@link #withSocketTimeout(long, TimeUnit)}
    */
+  @Deprecated(since = "9.2")
   public B withSocketTimeout(int socketTimeoutMillis) {
+    withSocketTimeout(connectionTimeoutMillis, TimeUnit.MILLISECONDS);
```

Comment:
you aren't referring to the parameter; this is why the test failed.  IntelliJ made this easy to spot; showed the parameter in gray.